src/utils/helper.js

```diff
@@ -1,19 +1,19 @@
 import { chainList, rpcLinks, scans, month } from "../config/constants";
 
-export const assetName = (name) => {
-  return name ? name : 'Off Chain Asset';
-}
-
 export const shortAddress = (addr) => {
-  return addr?.slice(0, 6) + '...' + addr?.slice(-4)
+  if(!addr) return
+  return `${addr.slice(0, 6)} ··· ${addr.slice(-4)}`
 }
 
-export const scanLink = (chainId, addr) => {
-  return scans[chainId] + addr;
+export const scanLink = (network, addr) => {
+  const chain = chainList.filter(net => net.name.toLowerCase() === network);
+  return scans[chain[0]?.chainId] + addr;

```

Comment:
did you make 0 (mainnet) the default? the default should be always the governance chain, arbitrum

Reply:
no, I used a filter function. This return an array

Reply:
You're right, then let's use `find` instead of `filter` as it will return 1 instance or null